sdk/eventhub/event-hubs/test/internal/sender.spec.ts

```diff
@@ -976,7 +977,7 @@ describe("EventHub Sender", function(): void {
         throw new Error(`Test failure`);
       } catch (err) {
         err.name.should.equal("AbortError");
-        err.message.should.equal("The send operation has been cancelled by the user.");
+        err.message.should.equal("Send request has been cancelled.");
```

Comment:
Is this message different because it comes from rhea-promise?

Reply:
yes :(